cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java

```diff
@@ -74,6 +74,10 @@ public void destroy() {
     }
   }
 
+  public String apiUrlPrefix() {
+    return _config.getString(WebServerConfig.WEBSERVER_API_URLPREFIX_CONFIG);
+  }
+
```

Comment:
Was this added intentionally?

Reply:
I was added for testing purposes. I'll remove it.